tests/test-dladdr.c

```diff
@@ -138,7 +140,7 @@ int main(int argc, char **argv)
     /* -ldl */
     result |= check_dladdr( (void*)dlopen, "dlopen", Pass );
     /* -lglibc */
-    result |= check_dladdr( (void*)atoi, "atoi", Pass );
+    result |= check_dladdr( (void*)atoi, "atoi", PassWithDifferentAddressOnWin );
```

Comment:
Why it has different address?

Reply:
what about trying to add `__declspec(dllimport) int atoi(const char *str);` into `#ifdef _WIN32` section?

Reply:
> Why it has different address?

Because it was imported from msvcrt

 `check address 004031BC with has symbol 'atoi' -> passed (filename: C:\windows\system32\msvcrt.dll base: 7ED70000 symbol name: 'atoi' addr: 7EDCC790)`

Reply:
So then adding above dllimport line should help.

Reply:
`/usr/i686-w64-mingw32/sys-root/mingw/include/stdlib.h:428:  int __cdecl atoi(const char *_Str);`

Reply:
so then try to put following line after `#ifdef _WIN32` section:
```c
__declspec(dllimport) int __cdecl atoi(const char *_Str);
```
it should override previous definitions.

Reply:
So have you tried to add that line into #idef _WIN32 section?

Reply:
That change has been dropped.

Reply:
Seems that `PassWithDifferentAddressOnWin` (different address) is still there and was not dropped. Or do you mean something else?